stage/unstable/debian/testing/package-model.json

```diff
@@ -9,7 +9,10 @@
       "ref": "fix/trixie-session-init-1094494",
       "source": "https://github.com/regolith-linux/regolith-session.git"
     },
-    "sway-regolith": null,
+    "sway-regolith": {
+      "ref": "feat/v1.10-port",
```

Comment:
If this is only temporary to build. You can leave the PR alone and build it via workflow. Otherwise I suggest to change this to `packaging/v1.10-regolith`